src/main/java/org/reaktivity/specification/socks/internal/Functions.java

```diff
@@ -0,0 +1,78 @@
+/**
```

Comment:
This class will no longer be needed once you remove all writing of tcp begin ex.